### ISSUE TITLE
Backport support for python 3.10.2? (conda-forge)

### ISSUE THREAD

**Issue by emirkmo** [astropy/regions#432]
Is it possible to backport support for python 3.10.2 to regions v0.5.x on conda?  

If I try a fresh install on conda-forge channel (recommended in the docs) with latest python and regions, it fails to install:

```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - regions -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.5,<3.6.0a0']

Your python: python=3.10.2
```

Edit: Actually pip install regions works in this case. So it's a conda packaging issue.

**Comment by jzuhone** [comment]
I'd love to see this also, as regions is a dependency for https://github.com/lynx-x-ray-observatory/soxs

**Comment by larrybradley** [comment]
`conda-forge` will not build `regions v0.5` for Python 3.10 because of it's support for `numpy 1.16` (see https://conda-forge.org/status/#python310):

<img width="779" alt="regions" src="https://user-images.githubusercontent.com/4992897/155204384-3d6db3b9-9e0b-4279-8995-7acac012ac39.png">

Unfortunately, conda-forge python 3.10 builds will not be available until regions 0.6, which will require `numpy >= 1.18` and `Python >= 3.8`.  The good news is that I am planning to release regions 0.6 within the next two weeks.

I also just uploaded python 3.10 wheels to PyPI.

**Comment by emirkmo** [comment]
Good news on the horizon then. Regions is a super helpful package. Thanks for the wheels.
